edge/index.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
I think it would be best to host source code in this repo.
We will have to set up hosting and repo metadata building job anyway so by having just source files here we'll save ourselves time by not having to package things.

Reply:
done!